solutions/devsprint-firas-safa-1/GitHubApp/Screens/List/ListView.swift

```diff
@@ -8,29 +8,27 @@
 import UIKit
 
 struct ListViewConfiguration {
-
     let listItems: [String]
 }
 
 final class ListView: UIView {
-
-    private let listViewCellIdentifier = "ListViewCellIdentifier"
-
-    private var listItems: [String] = []
+    var viewModel: ListViewModel {
+        didSet { tableView.reloadData() }
+    }
```

Comment:
Create a function called `update`, to which we pass the viewmodel, and call it in the did set. Inside of that funciton call reloadData. The idea is if in thefuture we want to do more stuff when the viewmodel is updated, we already have a funciton implemented and just need to add stuff there.